### DISCUSSION TITLE
Privileged containers and kernel modules

### DISCUSSION THREAD

**Discussion by adyanth** [gradescope/autograder_samples#111] [Q&A]
I would like to `modprobe` certain kernel modules and run the autograder container with `--privileged` to be able to run mininet and certain other software. Is this not possible at all with Gradescope?

Any updates since discussion https://github.com/gradescope/autograder_samples/discussions/86 ?

**Comment by adam-brucker** [comment]
Unfortunately the answer in #86 is still the same.  Thanks!
